store/src/java/com/zimbra/cs/service/admin/GetDomainInfo.java

```diff
@@ -129,8 +133,13 @@ private void addAttrElementIfNotNull(Entry entry, Element element, String key) t
                 element.addNonUniqueElement(AdminConstants.E_A).addAttribute(AdminConstants.A_N, key).setText(value);
             }
         } else {
+            // checking whether Modern UI is installed
+            File modernUIFile = new File(modernUIFolderPath);
             String value = entry.getAttr(key, null);
-            if (value != null) {
+            if (key.equals(ZAttrProvisioning.A_zimbraModernWebClientDisabled) && !(modernUIFile.exists() && modernUIFile.isDirectory())) {
+                element.addNonUniqueElement(AdminConstants.E_A).addAttribute(AdminConstants.A_N, key).setText(TRUE);
```

Comment:
If this was an existing constant for boolean values in XML attributes, we should probably keep using it, to properly transfer the canonical string representation, e.g. "TRUE" instead of "true" or "t" or "yes" or "1" or whatever. I recall using these TRUE/FALSE constants somewhere in Provisioning...